simexpal/launch/common.py

```diff
@@ -6,10 +6,13 @@
 import time
 
 import yaml
+import warnings
 
 from .. import base
 from .. import util
 
+warnings.simplefilter(action='default', category=DeprecationWarning)  # do not ignore DeprecationWarnings
```

Comment:
When would the warning be ignored otherwise? IMHO, simexpal should not mess with global warning filters (because it can also be used as a library).

Reply:
The ``warnings`` module ignores ``DeprecationWarnings`` by default. Thats why I set ``action='default'``, which prints the first occurrence of the warning in each module. I actually set this warnings filter in ``simexpal/base.py`` already, so I might delete this line in ``simexpal/launch/common.py``

Reply:
Hm, my point still stands: importing a library should not change global warning settings. We can set the filter in the `simex` script since in that script, we control the full environment.

Reply:
And it should also be removed from `launch/common.py`.

Reply:
Oh, I agree. I somehow overlooked the possibility to set the filter in the ``simex`` script. I will remove the line in the other files.